src/Controller/CRUDController.php

```diff
@@ -52,6 +52,11 @@ class CRUDController implements ContainerAwareInterface
         ControllerTrait::render as originalRender;
     }
 
+    /**
+     * @var ContainerInterface
+     */
+    protected $container;
```

Comment:
Why is this needed?

Reply:
because PHPstan doesn't understand that it's symfony container.
It thinks it's PrsContainer. 
Because `CRUDController` uses `ControllerTrait` which has 
```/**
 * @property ContainerInterface $container
 */
```
property annotation and it picks it up.